tests/test_factory_constructors.py

```diff
@@ -101,6 +104,17 @@ def test_init_factory_signature(msg):
     """  # noqa: E501 line too long
     )
 
+    assert (
+        msg(m.TestFactory2.__init__.__doc__)
+        == """
+        __init__(self: m.factory_constructors.TestFactory2, arg0: m.factory_constructors.tag.pointer_tag, arg1: int) -> None \\
+        __init__(self: m.factory_constructors.TestFactory2, arg0: m.factory_constructors.tag.unique_ptr_tag, arg1: str) -> None \\
+        __init__(self: m.factory_constructors.TestFactory2, arg0: m.factory_constructors.tag.move_tag) -> None
+        This is one part of the docstring.
```

Comment:
So, as said above, I think I'd like to see a single white line above this. Wouldn't that look better, splitting off the signatures from the body?

Anyway, just pointing out in the tests, which seems easier to judge.